### PR TITLE
Added roadrunner-ubiquity Integration to the devtools

### PR DESCRIPTION
**Pull request by wizzybenson**

### CODE REVIEW

File: src/Ubiquity [hunk 1]
```diff
@@ -85,7 +85,12 @@ class Ubiquity {
 			'checkClass' => '\\Ubiquity\\servers\\workerman\\WorkermanServer',
 			'require' => 'phpmv/ubiquity-workerman',
 			'start' => '_workerman.php start'
-		]
+		],
+        'roadrunner' => [
+            'files' => [],
```
Comment (jcheron) [inline src/Ubiquity]: missing:
```
'checkClass' => '\\RoadRunnerUbiquity\\Request'
```

File: src/Ubiquity [hunk 2]
```diff
@@ -810,6 +815,17 @@ class Ubiquity {
 				echo ConsoleFormatter::showMessage($msg . "Use <b>Ubiquity serve:stop</b> {$type} to stop it!\n");
 				system('php .ubiquity' . \DS . $serverInfos['start']);
 				break;
+            case 'roadrunner':
+                if (! \file_exists("vendor/lapinskas/roadrunner-ubiquity")) {
+                    system('composer require ' . $serverInfos['require']);
+                }
```
Comment (jcheron) [inline src/Ubiquity]: To be removed if **checkClass** is filled in

**Comment by wizzybenson** [comment]
Hi Jcheron, Changes have been added. Thanks for the feedback

File: src/Ubiquity (hunk 1 above)
Comment (wizzybenson) [reply src/Ubiquity]: Has been added.

File: src/Ubiquity (hunk 2 above)
Comment (wizzybenson) [reply src/Ubiquity]: Done

**Pull request merged**